src/main/java/com/es/jointexpensetracker/model/Expense.java

```diff
@@ -17,8 +17,12 @@ public Expense(Long id, String description, BigDecimal amount, String person, St
         this(id, description, amount, Currency.getInstance("USD"), person, LocalDate.now(), expenseGroup);
     }
 
+    public Expense() {
+        this(null, null, null, null, null, null, null);
```

Comment:
this line does nothing. All the fields are initialized with nulls by default.
Remove the line.